src/seedu/addressbook/AddressBook.java

```diff
@@ -1155,8 +1155,9 @@ private static String getUsageInfoForExitCommand() {
      * @param sign  Parameter sign to be removed
      * @return  string without the sign
      */
-    private static String removePrefixSign(String s, String sign) {
-        return s.replace(sign, "");
+    public static String removePrefix(String fullString, String prefix) {
+        String prefixWithRegex = "^" + prefix;
+		return fullString.replaceFirst(prefixWithRegex, "");
```

Comment:
indentation is off here!
our coding standards require use of 4 spaces for indentation
Tip: You can get eclipse to do this for you, check out how!